autoPyTorch/pipeline/components/training/data_loader/image_data_loader.py

```diff
@@ -63,5 +63,5 @@ def _check_transform_requirements(self, X: Dict[str, Any], y: Any = None) -> Non
         if not X['image_augmenter'] and 'image_augmenter' not in X:
             raise ValueError("Cannot find the image_augmenter in the fit dictionary")
 
-        if not X['dataset_properties']['is_small_preprocess'] and 'preprocess_transforms' not in X:
-            raise ValueError("Cannot find the preprocess_transforms in the fit dictionary")
+        # if not X['dataset_properties']['is_small_preprocess'] and 'preprocess_transforms' not in X:
```

Comment:
Can we continue this test by just removing `not X['dataset_properties']['is_small_preprocess']`?